src/main/resources/frameweb-tools.properties

```diff
@@ -10,3 +10,5 @@ default.entity.attribute.constraint.generation = generation\=auto
 default.entity.generalization.stereotype = single-table
 default.entity.association.end.constraint.collection = collection\=set
 default.entity.association.end.constraint.cascade = cascade=none
+default.generate.code.output.folder = src/main/resources/generated
```

Comment:
```suggestion
default.generate.code.output.folder = target/
```

If I understood this well, this is what comes filled in the code generation dialog, right? I suggest something simpler as above, or maybe `generated/`, also to avoid looking like FrameWeb is for Java/Maven only.

Reply:
Fixed: Also, removed the template default referring to the resources folder.